Workbooks/Azure Backup/Backup Audit Reporting Template/Backup Audit Reporting Templates.workbook

```diff
@@ -258,171 +258,183 @@
-              "size": 3,
-              "queryType": 8,
-              "visualization": "tiles",
-              "gridSettings": {
-                "formatters": [
-                  {
+              "type": 3,
+              "content": {
+                "version": "KqlItem/1.0",
+                "query": "{\"version\":\"1.0.0\",\"content\":\"[{\\\"Name\\\": \\\"Backup Configuration Status\\\", \\\"Description\\\":\\\"View which VMs haven't been configured for backup\\\",\\\"Value\\\":10, \\\"URL\\\":\\\"Community-Workbooks/Azure Backup/Backup Configuration Status\\\",\\\"LearnMoreLink\\\":\\\"https://go.microsoft.com/fwlink/?linkid=2211462&clcid=0x409\\\"},\\r\\n{\\\"Name\\\": \\\"Backup job history\\\", \\\"Description\\\":\\\"View information on backup jobs over a period of time and their associated status\\\",\\\"Value\\\":10, \\\"URL\\\":\\\"Community-Workbooks/Azure Backup/Backup Job History\\\",\\\"LearnMoreLink\\\":\\\"https://go.microsoft.com/fwlink/?linkid=2211463&clcid=0x409\\\"},\\r\\n{\\\"Name\\\": \\\"Backup schedule and retention\\\", \\\"Description\\\":\\\"Check if all your backed up items have the required backup schedule and retention\\\",\\\"Value\\\":10, \\\"URL\\\":\\\"Community-Workbooks/Azure Backup/Backup Schedule and Retention\\\",\\\"LearnMoreLink\\\":\\\"https://go.microsoft.com/fwlink/?linkid=2211542&clcid=0x409\\\"},\\r\\n{\\\"Name\\\": \\\"User triggered operations\\\", \\\"Description\\\":\\\"View information on user triggered operations like configuration changes and restores\\\",\\\"Value\\\":10, \\\"URL\\\":\\\"Community-Workbooks/Azure Backup/User Triggered Operations\\\",\\\"LearnMoreLink\\\":\\\"https://go.microsoft.com/fwlink/?linkid=2211543&clcid=0x409\\\"},\\r\\n{\\\"Name\\\": \\\"ASR Job History\\\", \\\"Description\\\":\\\"View information on the status of ASR jobs over a specified duration of time \\\",\\\"Value\\\":10, \\\"URL\\\":\\\"Community-Workbooks/Azure Backup/ASR Job History\\\",\\\"LearnMoreLink\\\":\\\"https://go.microsoft.com/fwlink/?linkid=2211543&clcid=0x409\\\"},\\r\\n{\\\"Name\\\": \\\"ASR Replication History\\\", \\\"Description\\\":\\\"View information on health of replicated items over a specified duration of time\\\",\\\"Value\\\":10, \\\"URL\\\":\\\"Community-Workbooks/Azure Backup/ASR Replication History\\\",\\\"LearnMoreLink\\\":\\\"https://go.microsoft.com/fwlink/?linkid=2211543&clcid=0x409\\\"}]\",\"transformers\":null}",
```

Comment:
Instead of Community-Workbooks/Azure Backup/ASR Replication History it will be ''Community-Workbooks/Azure Backup/ASR ReplicatedItems''